speech/google/cloud/speech/sample.py

```diff
@@ -112,3 +117,180 @@ def encoding(self):
         :returns: String value of Encoding type.
         """
         return self._encoding
+
+    def async_recognize(self, language_code=None, max_alternatives=None,
+                        profanity_filter=None, speech_context=None):
+        """Asychronous Recognize request to Google Speech API.
```

Comment:
1. Yeah I just copied them over and deleted references to the `sample` param.
2. I could, however, those settings don't really have to do with the `sample` so much as they do the request. Semantics but should I feel like those settings belong in `Sample`?